apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -21,15 +22,20 @@ const getUserIdentify = () => {
 
 const UseDeskWidget: React.FC = () => {
     const { link } = useOrganization()
+    const { user, refetch } = useAuth()
+
+    useEffect(() => {
+        refetch()
```

Comment:
It looks strange and hacky. Why the widget should refer him the user?

Reply:
the widget does not send the user. It sends out the phone and mail, getting them from the user

Reply:
why you need to use refresh here? it's already inside the useAuth. Why you need to rerender whole page ?

Reply:
good question :) corrected now :)